build/build-image/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2014 Google Inc. All rights reserved.
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
Please don't bump the copyright on the file unless it's been basically rewritten (it hasn't).